operational/azure/azure_long_running_instances/azure_long_running_instances.pt

```diff
@@ -0,0 +1,281 @@
+name "Azure Long Running Instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks for running instances that have been running longer than the `Days Old` parameter. See the [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "low"
+category "Operational"
+info(
+    version: "2.0",
+    provider: "Azure",
+    service: "Compute",
+    policy_set: "Long Running Instances"
+)
+
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+    type "list"
+    label "Email notify list"
+    description "Email addresses of the recipients you wish to notify when new incidents are created"
+end
+
+parameter "param_days_old" do
+    type "number"
+    label "Days Old"
+    description "Number of days to be running before included in list"
+    default 180
+  end
+
+  parameter "param_automatic_action" do
+    type "string"
+    label "Automatic Actions"
+    description "When this value is set, this policy will automatically take the selected action(s)"
+    default "Select Action"
+    allowed_values "Select Action", "Stop Instances", "Terminate Instances"
+  end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+#authenticate with Azure
+credentials "auth_azure" do
```

Comment:
Do this credentials have permissions to stop and delete instances at Azure?

Reply:
Hello! 

To be honest, even though I was applying the policy in test mode, I wasn't sure if the credentials had the permissions to do that and didn't want to test that much and accidentally delete or turn off some vm's in production.

I would love to get some feedback about that.

Reply:
Got it! I'm pretty sure it'll work!